tests/php/test_class_wc-connect-nux.php

```diff
@@ -7,56 +7,91 @@ public static function setupBeforeClass() {
 	}
 
 	public function test_get_banner_type_to_display_dev_jp() {
-		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
-			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_DEV,
-			'tos_accepted'                           => false,
-			'should_display_after_cxn_banner'        => false,
-		) ), false );
-
-		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
-			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_DEV,
-			'tos_accepted'                           => true,
-			'should_display_after_cxn_banner'        => false,
-		) ), false );
-
-		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
-			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_DEV,
-			'tos_accepted'                           => false,
-			'should_display_after_cxn_banner'        =>  true,
-		) ), false );
-
-		$this->assertEquals( WC_Connect_Nux::get_banner_type_to_display( array(
-			'jetpack_connection_status'              => WC_Connect_Nux::JETPACK_DEV,
-			'tos_accepted'                           => true,
-			'should_display_after_cxn_banner'        =>  true,
-		) ), false );
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => false,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => false,
+				)
+			), false
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => true,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => false,
+				)
+			), false
+		);
+
+		$this->assertEquals(
+			WC_Connect_Nux::get_banner_type_to_display(
+				array(
+					'jetpack_connection_status'       => WC_Connect_Nux::JETPACK_DEV,
+					'tos_accepted'                    => false,
+					'can_accept_tos'                  => false,
+					'should_display_after_cxn_banner' => true,
+				)
+			), 'after_jetpack_connection'
```

Comment:
Should we be displaying the 'after_jetpack_connection' banner if the user can't accept TOS?